cypress.json

```diff
@@ -7,5 +7,6 @@
     "supportFile": "src/test/cypress/support/index.ts",
     "baseUrl": "http://localhost:8080",
     "video": false,
-    "screenshotOnRunFailure": false
+    "screenshotOnRunFailure": false,
+    "chromeWebSecurity": false
```

Comment:
Would be great if we could avoid using that flag

Reply:
Fair point i forgot about that. ill try to fix it.